edx_name_affirmation/migrations/0004_remove_verifiedname_is_verified.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.24 on 2021-08-13 13:07
```

Comment:
Please do not check in this migration file at the same time as this PR. The migration file should be deployed later. According to [HowToDropAColumn](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtodropacolumn) article, this should be done in at least 2 releases.